web/packages/teleterm/src/ui/DocumentTerminal/useDocumentTerminal.ts

```diff
@@ -42,13 +42,13 @@ export function useDocumentTerminal(doc: types.DocumentTerminal) {
   const logger = useRef(new Logger('useDocumentTerminal'));
   const ctx = useAppContext();
   const { documentsService } = useWorkspaceContext();
-  const [attempt, startTerminal] = useAsync(async () => {
+  const [attempt, runAttempt] = useAsync(async () => {
     if ('status' in doc) {
       documentsService.update(doc.uri, { status: 'connecting' });
     }
 
     try {
-      return await startTerminalSession(
+      return await initializePtyProcess(
```

Comment:
I tried to refactor the names a little bit to be more clear as to what is happening. `useDocumentTerminal` doesn't actually start the terminal session, it makes the RPC to intantiate a new PtyProcess in the shared process:

https://github.com/gravitational/teleport/blob/3588d98fd80dc1b5b6d746afa4cc76b73e486fde/web/packages/teleterm/src/sharedProcess/ptyHost/ptyHostService.ts#L31-L41

The start of the process, which actually spawns the process, is done from `ctrl.open`.